config/plop/templates/block/Block.astro.hbs

```diff
@@ -1,7 +1,7 @@
 ---
 import type { {{ pascalCase name }}Fragment } from '@lib/types/datocms';
 
-interface Props {
+export interface Props {
```

Comment:
Astro does not support named exports in components. So this shouldn't work. Does it? The exported interfaces are not imported anywhere else right? So I wouldn't do this. If it works, it is undocumented. See https://docs.astro.build/en/basics/astro-components/#component-props